google/cloud/bigtable/table.py

```diff
@@ -782,6 +784,180 @@ def mutations_batcher(self, flush_count=FLUSH_COUNT, max_row_bytes=MAX_ROW_BYTES
         """
         return MutationsBatcher(self, flush_count, max_row_bytes)
 
+    def backup(self, backup_id, cluster_id=None, expire_time=None):
+        """Factory to create a Backup linked to this Table.
+
+        :type backup_id: str
+        :param backup_id: The ID of the Backup to be created.
+
+        :type cluster_id: str
+        :param cluster_id: (Optional) The ID of the Cluster. Required for
```

Comment:
Seems odd we have an optional but required parameter. Can we get this by retrieving the backup? This kind of seems required? What is the use case gained by making this optional? Is this just to make it easier to create a client side object?